Mytinerary/View Controllers/InputEventViewController.m

```diff
@@ -140,6 +149,8 @@ - (void)initActivityEvent {
         }
         else {
             NSLog(@"error initializing activity event: %@", error);
+            self.alert.message = error.domain;
+            [self presentViewController:self.alert animated:YES completion:nil];
```

Comment:
might want to combine this into its own function since you likely never want to alert without setting self.alert.message first.